pvr.vuplus/changelog.txt

```diff
@@ -1,6 +1,7 @@
 v3.17.1
 - Fixed: Ensure only one call to GetEPGForChannel happens at one time, fixes #181
 - Added: Update/fix server version for pvr addon
+- Added: Option to skip initial EPG Load
```

Comment:
Imo, functional extensions justify a minor version bump. micro bumps are for fixes only.